docs/api-references/device-lifecycle-management-api.md

```diff
@@ -0,0 +1,53 @@
+# Device Lifecycle Management API
+
+## 1 Card Status
+
+### 1.1 getRegistrationData
+
+This API is used to check the registration status of the card, i.e. if the card user is registered with CP on behalf of the program. For that, this API will fetch the registration data from the card which include the R-ID and the authentication method.
+
+```
+NOTE: It is the responsibility of the Reliant Application to show and capture the user’s consent.
+Then, the Reliant Application must store it with CPK.
+```
+
+**Compatibility**
+| **Available as of CPK version #** | **Deprecated as of CPK version #** |
+|--------------------------------------------------|------------------------------------------------------------------|
+| + CPK 2.0.1 | + n/a |
+
+**Input Parameters**
+| **Parameter** | **Type** | **Description** |
+|---------------|----------|------------------------------------------------------|
+| getRegistrationDataRequest | GetRegistrationDataParamType | An object that contains a reliantGUID and programGUID |
+
+**Response Parameters**
+| **Parameter** | **Type** | **Description** |
+|-----------------|-----------------|----------------------------------------------------------|
+| getRegistrationDataResponse | Promise<`GetRegistrationDataResultType`> | A promise that resolves to an object containing either an authMethods array with a boolean field indicating whether the user is registered in the current program, or an error field. |
+
+**Type Aliases**
+
+```ts
+// SaveBiometricConsentParamType
+interface GetRegistrationDataParamType {
+  reliantGUID: string;
+  programGUID: string;
+}
+
+// SaveBiometricConsentResultType
```

Comment:
Please update this to: // GetRegistrationDataResultType

Reply:
Thanks, Corrected